src/clang-c-frontend/clang_c_convert.cpp

```diff
@@ -1967,40 +1956,37 @@ bool clang_c_convertert::get_expr(const clang::Stmt &stmt, exprt &new_expr)
 
     exprt inits;
 
-    // the type might contains symbolic types,
-    // replace them with complete types before getting the initializations
-    typet complete_type;
-    bool contains_symbolic =
-      contains_symbolic_struct_types(t, complete_type, ns);
-    if(contains_symbolic)
-      t = complete_type;
+    const typet &t_ref = ns.follow(t);
 
     // Structs/unions/arrays put the initializer on operands
-    if(t.is_struct() || t.is_array() || t.is_vector())
+    if(t_ref.is_struct() || t.is_array() || t.is_vector())
```

Comment:
t_ref for array and vector as well?

Reply:
I've removed t_ref and updated t directly.